docs/brandenberg/usecase.md

```diff
@@ -1,89 +1,893 @@
 
-| Column 1 | Column 2 | Column 3 |
-|----------|----------|----------|
-| Stampede2| CPU      | 2017     |     
-| Frontera | CPU & GPU| 2019     |     
+![Screenshot of result of query of Wildlife liquefaction array query of event information and field observations.](img/WildlifeQuery1.png)
+  
 
-Or use markdown table generator: [https://www.tablesgenerator.com/markdown_tables](https://www.tablesgenerator.com/markdown_tables)
+**Figure 2.** Screenshot of result of query of Wildlife liquefaction array query of event information and field observations.
 
+### Query Wildlife liquefaction CPT data
 
-### Math
+This query retrieves all cone penetration test data from the Wildlife liquefaction array. INNER JOIN statements are needed to link SCPT to SCPG (using SCPG_ID), SCPG to TEST (using TEST_ID), and TEST to SITE (using SITE_ID). This query demonstrates propagation of primary and foreign keys through the schema heirarchy.
 
-To generate math equations in markdown.
+```python  
+import pandas as pd
+import designsafe_db.ngl_db as ngl
 
-For inline mode formulas: $`a^2+b^2=c^2`$.
+command = 'SELECT TEST.TEST_ID, TEST.TEST_NAME, SCPT. SCPT_DPTH, SCPT.SCPT_RES, SCPT.SCPT_FRES FROM SCPT '
+command += 'INNER JOIN SCPG ON SCPT.SCPG_ID = SCPG.SCPG_ID '
+command += 'INNER JOIN TEST ON TEST.TEST_ID = SCPG.TEST_ID '
+command += 'INNER JOIN SITE ON SITE.SITE_ID = TEST.SITE_ID '
+command += 'WHERE SITE.SITE_NAME = "Wildlife Array"'
 
-For display mode formulas which appear on a separate line
-```math
-f(x) = \int_{-\infty}^\infty
-\hat f(\xi)\,e^{2 \pi i \xi x}
-\,d\xi
+df = ngl.read_sql(command)
+pd.set_option('display.max_rows', 10)
+df
 ```
 
+![Screenshot of result of query of Wildlife liquefaction array query of cone penetration test data.](img/WildlifeQuery2.png)
+
+**Figure 3.** Screenshot of result of query of Wildlife liquefaction array query of cone penetration test data.
+
+### Query number of data entries in various tables, including indication of review status
+
+This query builds upon the previous query by adding an indication of whether the data quantity has been reviewed. Data in the NGL database is submitted for review by users, and subsequently reviewed by members of the database working group to check the data against published sources, identify errors, and ensure data entry completeness.
+
+```python
+import designsafe_db.ngl_db as ngl
+import pandas as pd
+```
+
+command = 'SELECT (SELECT COUNT(SCPG_ID) FROM SCPG) as "CPT Soundings", '
+command += '(SELECT COUNT(BORH_ID) FROM BORH) as "Boreholes", '
+command += '(SELECT COUNT(GSWG_ID) FROM GSWG) as "Surface Wave Measurements", '
+command += '(SELECT COUNT(GINV_ID) FROM GINV) as "Invasive VS Measurements", '
+command += '(SELECT COUNT(FLDM_ID) FROM FLDM WHERE FLDM_SFEV=1) as "Liquefaction Observations", '
+command += '(SELECT COUNT(FLDM_ID) FROM FLDM WHERE FLDM_SFEV=0) as "Non-Liquefaction Observations"'
+
+scpg_count = ngl.read_sql(command).T
+scpg_count.columns=["Count"]
+scpg_count
```

Comment:
Add a code section using ```

Reply:
I have removed all code blocks from the documentation